src/tests/gc-e2e/GuestConfiguration.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# GuestConfiguration.Tests.ps1
```

Comment:
>GuestConfiguration [](http://example.com/codeflow?start=2&length=18)

Change the "GuestConfiguration" name everywhere. 

See the updated PR title.

This E2E test is for the Universal NRP via the Machine Configuration PS cmdlets.

So the correct name is: 

"UniversalNrp" instead of "GuestConfiguration"

Reply:
Got it, i can change that!